src/kvm_ioctls.rs

```diff
@@ -216,8 +216,10 @@ ioctl_ior_nr!(KVM_ARM_PREFERRED_TARGET, KVMIO, 0xaf, kvm_vcpu_init);
 
 /* Available with KVM_CAP_DEVICE_CTRL */
 ioctl_iowr_nr!(KVM_CREATE_DEVICE, KVMIO, 0xe0, kvm_create_device);
-/* Available with KVM_CAP_VM_ATTRIBUTES */
+/* Available with KVM_CAP_DEVICE_CTRL */
```

Comment:
Why change: 3 capabilities (`KVM_CAP_DEVICE_CTRL`, `KVM_CAP_VM_ATTRIBUTES `and `KVM_CAP_VCPU_ATTRIBUTES`) all have `KVM_SET/HAS_DEVICE_ATTR`, but currently we only support the one in DEVICE.